fpdf/encryption.py

```diff
@@ -137,17 +146,26 @@ def __init__(
         self.cf = None
         self.key_length = 128
 
+        if import_error and self.encryption_method in (
+            EncryptionMethod.AES_128,
+            EncryptionMethod.AES_256,
+        ):
+            raise EnvironmentError(
+                "cryptography module not available"
+                " - Try: 'pip install cryptography' or use RC4 encryption method"
+                f" - Import error was: {import_error}"
+            )
         if self.encryption_method == EncryptionMethod.AES_128:
-            if import_error:
-                raise EnvironmentError(
-                    "cryptography module not available"
-                    " - Try: 'pip install cryptography' or use RC4 encryption method"
-                    f" - Import error was: {import_error}"
-                )
             self.v = 4
             self.r = 4
             fpdf._set_min_pdf_version("1.6")
             self.cf = CryptFilter(mode="AESV2", length=self.key_length)
+        elif self.encryption_method == EncryptionMethod.AES_256:
+            self.v = 5
+            self.r = 6
```

Comment:
Could you maybe use enum values for those constants?
And maybe rename those two properties to make them more meaningful?
I currently do not really understand what they are used for.

Reply:
This QPDF document has the best explanation for the "R" and "V" values.
I'll rename them to "revision" and "version" on the module.